a3fe/run/simulation.py

```diff
@@ -537,55 +565,6 @@ def update_paths(self, old_sub_path: str, new_sub_path: str) -> None:
         """
         super().update_paths(old_sub_path, new_sub_path)
 
-        # Also need to update the slurm file base
-        if self.slurm_file_base:
```

Comment:
Correct to delete this, but let's make self.slurm_file_base a property which runs the slurm_config.get_file_base (or whatever it is) each time it's run, so that we don't need to store it (and it automatically responds to changes in the base dir)